debian/rules

```diff
@@ -4,7 +4,7 @@
 	dh $@
 
 override_dh_auto_configure:
-	./autogen.sh
+	dh_autoreconf
```

Comment:
dh-autoreconf ships with a dh sequencer. "dh $@ --with autoreconf" should do the trick.